tests/filefinder.cpp

```diff
@@ -6,33 +6,33 @@
 #include "main_data.h"
 
 namespace {
-/*
 	void CheckIsRPG2kProject() {
-		EASYRPG_SHARED_PTR<FileFinder::ProjectTree> const
-			tree = FileFinder::CreateProjectTree(".");
+		std::shared_ptr<FileFinder::DirectoryTree> const
+			tree = FileFinder::CreateDirectoryTree(".");
 		assert(FileFinder::IsRPG2kProject(*tree));
+		FileFinder::SetDirectoryTree(tree);
 	}
-*/
+
 	void CheckIsDirectory() {
 		assert(FileFinder::IsDirectory("."));
 	}
 
 	void CheckEnglishFilename() {
-		assert(!FileFinder::FindImage("Backdrop", "castle").empty());
```

Comment:
Well it searches for a file that has a English name :D. But I could change it to a japanese -> english lookup